benefits/static/css/styles.css

```diff
@@ -1,3 +1,28 @@
+/* making the sticky footer */
```

Comment:
How much of the CSS below is actually for the sticky footer, vs. just part of our design/layout? I worry about a comment like this indicating all of it related? Maybe it is.

Reply:
All of it up to line 25 is necessary for the sticky footer. Any ideas on how to make that more obvious? 🤔

Reply:
Hmm maybe I am expecting it to come after the more generic HTML element overrides - any reason it moved up?

Reply:
Maybe it's more about how we organize our CSS too (also, I thought we weren't worry about CSS until all the copy edits??)